etc/testing/images/ubuntu_with_s3_clients/Dockerfile

```diff
@@ -8,11 +8,12 @@ RUN apt-get update -y && apt-get install -y \
   python3-setuptools \
   groff \
   apt-transport-https \
-  ca-certificates gnupg
+  ca-certificates gnupg \
+  wget
 RUN python3 -m easy_install pip
 RUN pip3 install --upgrade pip
 RUN pip3 install awscli --upgrade
-RUN curl -L https://dl.min.io/client/mc/release/linux-amd64/mc >/usr/local/bin/mc \
+RUN wget -O /usr/local/bin/mc https://dl.min.io/client/mc/release/linux-amd64/mc \
```

Comment:
nit: Unless we need to know the download progress, I'd use -nv to silence all the download logs.

Reply:
thanks! fixed!